src/libp2p/websocketOverTor.test.ts

```diff
@@ -0,0 +1,199 @@
+import WebsocketsOverTor from './websocketOverTor'
+import connect from 'it-ws/client'
+import Multiaddr from 'multiaddr'
+import toConnection from 'libp2p-websockets/src/socket-to-conn'
+import { createServer } from 'it-ws'
+import url, { Url, UrlWithStringQuery } from 'url'
+
+jest.mock('it-ws/client')
```

Comment:
I think we mock too much.
I was actually looking to see if the connections are actually getting established, while here everything is mocked out